cacli/cmd/cancel.go

```diff
@@ -22,19 +22,19 @@ THE SOFTWARE.
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/cloud-annotations/training/cacli/cmd/cancel"
 	"github.com/spf13/cobra"
 )
 
 // cancelCmd represents the cancel command
 var cancelCmd = &cobra.Command{
 	Use:   "cancel <model-id>",
-	Short: "Coming soon",
-	Long:  `Coming soon.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("This command is not yet available.")
-	},
+	Short: "Cancel a training run",
+	Long: `Cancel a training run that is either pending or running. You can not cancel a training that is complete or already canceled of failed.
```

Comment:
Keep the line break where I had it in the suggestion. I want the actual text to break at 80 char when running `cacli cancel --help`.

Reply:
also you didn't fix the typo `of` -> `or` and use a comma instead of double `or`: `complete, canceled or failed`

and `training run` not `training`

Reply:
```
	Long: `Cancel a training run that is either running or pending. You can not cancel a
training run that is complete, canceled or failed.

Basic Example:
  cacli cancel MODEL-ID`,
```

Reply:
ensure that the space in front of `cacli cancel MODEL-ID` is 2 spaces not a tab.